examples/wgpu-example-runner/src/main.rs

```diff
@@ -126,8 +126,17 @@ async fn run(event_loop: EventLoop<()>, window: Window, swapchain_format: wgpu::
 }
 
 fn main() {
+    const WIDTH: u32 = 1280;
+    const HEIGHT: u32 = 720;
+
     let event_loop = EventLoop::new();
-    let window = winit::window::Window::new(&event_loop).unwrap();
+    let window = winit::window::WindowBuilder::new()
+        .with_inner_size(winit::dpi::LogicalSize::new(
+            f64::from(WIDTH),
```

Comment:
I think I would rather just put f64 literals here (`1280.0`) than defining separate u32 consts.

Reply:
Yup, I was mirroring `examples/example-runner-cpu`, but here we don't actually don't use the consts for anything other than initializing the window size.